osf/management/commands/migrate_deleted_date.py

```diff
@@ -0,0 +1,112 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection, transaction
+from framework.celery_tasks import app as celery_app
+from framework import sentry
+
+logger = logging.getLogger(__name__)
+
+DEBUG_PAGE_SIZE = 100000
```

Comment:
hmm, I agree this is probably fine most of the time, but this could leave data unmigrated - main objective for debug mode is to migrate all of it.  Brian's `migrate_pagecounter_data` solved it this way - 
```
LIMIT_CLAUSE = ' LIMIT %s);'
NO_LIMIT_CLAUSE = ');'
```